go/libkb/keyfamily.go

```diff
@@ -639,6 +639,7 @@ func (kf *KeyFamily) LocalDelegate(key GenericKey) (err error) {
 // account whether the key has been cancelled at time t.
 func (ckf ComputedKeyFamily) GetKeyRoleAtTime(kid keybase1.KID, t time.Time) (ret KeyRole) {
 	if info, err := ckf.getCkiIfActiveAtTime(kid, t); err != nil {
+		ckf.G().Log.Debug("GetKeyRoleAtTime %s, %s => err %s", kid, t, err)
```

Comment:
Yeah, I think it's helpful...helped me find this issue.  But if too verbose, I can get rid of it.